cmd/identity/certificate_authority.go

```diff
@@ -61,3 +79,36 @@ func cmdGetID(cmd *cobra.Command, args []string) (err error) {
 	fmt.Println(p.ID.String())
 	return nil
 }
+
+func cmdRevokeCA(cmd *cobra.Command, args []string) (err error) {
+	ca, err := revokeCACfg.CA.Load()
+	if err != nil {
+		return err
+	}
+
+	// NB: backup original cert
+	var backupCfg provider.FullCAConfig
+	extRegex, err := regexp.Compile(filepath.Ext(revokeCACfg.CA.CertPath) + "$")
+	if err != nil {
+		return err
+	}
+	backupCfg.CertPath = extRegex.ReplaceAllString(
+		revokeCACfg.Identity.CertPath,
+		strconv.Itoa(int(time.Now().Unix()))+".bak$1",
+	)
+	if err := backupCfg.Save(ca); err != nil {
```

Comment:
Do you want to overwrite key here as well?

Reply:
Ah, noticed below that it doesn't overwrite anymore.

Reply:
no: https://github.com/storj/storj/pull/848/files#diff-548b0434be6594cc8a5c66fe4bc2793eR221